### ISSUE TITLE
SEO Support

### ISSUE THREAD

**Issue by prvn3294** [W3Layouts/gatsby-starter-delog#5]
Would love if we can add SEO component in the theme. #

**Comment by inconis** [comment]
Sorry have you upload the wesite on github pages ?
Can you see the picture ?

**Comment by prvn3294** [comment]
> Sorry have you upload the wesite on github pages ?
> Can you see the picture ?

Yes, using netlify it's working fine.

**Comment by inconis** [comment]
Ok thanks, because I'm having problems with github so I asked for this reason

**Comment by munwr** [comment]
> Would love if we can add SEO component in the theme. #

Hi @prvn3294,

Do you any suggestions in mind. As of now, I have only added option ad meta description

**Comment by munwr** [comment]
@prvn3294 yeah, sure that i understand. I was asking about specific SEO features you want to be added in delog
